MacOS import error during Tauri Action

```
error[E0432]: unresolved import `notify::ReadDirectoryChangesWatcher`
  --> src/main.rs:11:44
   |
11 | use notify::{recommended_watcher, Watcher, ReadDirectoryChangesWatcher};
   |                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `ReadDirectoryChangesWatcher` in the root
   |
help: consider importing this variant instead
   |
11 | use notify::{recommended_watcher, Watcher, notify::WatcherKind::ReadDirectoryChangesWatcher;
   |                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

warning: unused import: `Watcher`
  --> src/main.rs:11:35
   |
11 | use notify::{recommended_watcher, Watcher, ReadDirectoryChangesWatcher};
   |                                   ^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0432`.
warning: `komishon` (bin "komishon") generated 1 warning
error: could not compile `komishon` due to previous error; 1 warning emitted
       Error failed to build app: failed to build app
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command failed with exit code 1: yarn tauri build

```

--- a/src-tauri/src/main.rs
+++ b/src-tauri/src/main.rs
@@ -8,7 +8,8 @@ mod helpers;
 
 use std::path::Path;
 use std::sync::Mutex;
-use notify::{recommended_watcher, ReadDirectoryChangesWatcher};
+use notify::recommended_watcher;
+use notify::ReadDirectoryChangesWatcher;
 use models::deadline::{add_deadline_to_file, remove_deadline_from_file, update_file_deadline};
 use models::tag::{add_tag_to_file, remove_tag_from_file};
 use models::file::{get_base_dirs, base_dirs_vec, get_children_of, walk_and_save, remove_invalid_files_from_db, search_by_name, update_favorite_status, fetch_files_with_deadlines, fetch_favorited_files, fetch_single_file};

